timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -76,7 +77,7 @@ def __iter__(self) -> Iterator[Dict[str, Any]]:
             df = self.target_df.loc[item_id]
             time_series = {
                 FieldName.ITEM_ID: item_id,
-                FieldName.TARGET: df.squeeze().to_numpy(dtype=self.float_dtype),
```

Comment:
Is this a performance related change?

Reply:
Current code may lead to a bug. If `df` has length 1, then `df.squeeze()` will be a `float` instead of `pd.Series`. There is no clean way to convert a DataFrame to a Series (`squeeze` is the only function recommended for this purpose), so we rather do the flattening into 1D after converting to numpy.